src/js/view/controls/rightclick.js

```diff
@@ -33,8 +34,10 @@ export default class RightClick {
         var versionMeta = semverParts[1];
```

Comment:
We want to remove `versionMeta` from all prerelease players. There is no need to look for the word 'beta'. That was probably a miscommunication in the ticket requirements.

Just remove the use of  `isPrerelease` and `versionMeta` completely.

Reply:
For more clarity see the definition of "pre-release" at http://semver.org/

Reply:
@robwalch Ah, thanks! Removed the variables and the conditionals.